PhpAmqpLib/Helper/DebugHelper.php

```diff
@@ -101,6 +107,6 @@ public function debug_connection_start($version_major, $version_minor, $server_p
      * @param string $s
      */
     protected function print_msg($s) {
-        echo $s . PHP_EOL;
+        fprintf($this->debug_output, $s . PHP_EOL);
```

Comment:
`fprintf` expects a format string and arguments to be provided, so the semantics is different from `echo` (in presence of `%` symbol). Should it be a `fwrite` instead?